docs/products/opensearch/howto/opensearch-log-integration.rst

```diff
@@ -8,27 +8,39 @@ In this article, you will understand how to enable, edit and disable the logs in
 Enable logs integration
 -----------------------
 
-Here are the steps needed to enable logs integration. This allows you to send logs to your Aiven for OpenSearch from another Aiven service.
+Here are the steps needed to enable logs integration. This allows you to send your service's logs to your Aiven for OpenSearch from another Aiven service.
 
 1. Log in to the `Aiven web console <https://console.aiven.io/>`_.
 
-2. On the **Services** page, open the service you want to send the logs from.
+2. On the **Services** page, scroll down and choose **Service integrations**.
 
-3. On the **Logs** tab, click on **Enable Logs integration**. 
+3. On the **Logs** tab, click on **Use Integration**. 
 
-4. Choose an existing OpenSearch instance or create a new one, then select **Continue**.
+4. Select an existing OpenSearch instance or create a new one, then select **Continue**.
 
-5. Configure your ``index prefix`` and ``retention limit`` parameters, then select **Enable**
+5. Configure your ``index prefix`` and ``index retention limit`` parameters, then select **Enable**
 
 .. note::
-    If you want to effectively disable the index ``retention limit``, you can set it to the maximum value which is 10000 days.
+    If you want to effectively disable the ``index retention limit``, you can set it to the maximum value which is 10000 days.
 
 Your Aiven service is now sending logs to your OpenSearch service which you can explore further.
 
+Logs integration configuration
+------------------------------
+There are two parameters that you can adjust when integrating logs to your OpenSearch service:
+
+* ``index prefix``, specifies the prefix part.
+* ``index retention limit``, number of days to preserve the daily indexes.
+
+.. warning::
+    
+    The service's logs are sent from the selected service to your OpenSearch cluster. When the ``index retention limit`` is reached, those indexes are deleted from the OpenSearch cluster.
+
+
 Change logs integration configuration
--------------------------------------
+'''''''''''''''''''''''''''''''''''''
 
-You can change the configuration of the ``index prefix`` and ``retention limit`` after the integration is enabled.
+You can change the configuration of the ``index prefix`` and ``index retention limit`` after the integration is enabled.
 
 1. Log in to the `Aiven web console <https://console.aiven.io/>`_.
```

Comment:
What is this line doing here?

Reply:
I can remove it but I have added the first step from all of them sections.